faust/stores/rocksdb.py

```diff
@@ -393,7 +393,7 @@ def _dbs_for_key(self, key: bytes) -> Iterable[DB]:
 
     def _dbs_for_actives(self) -> Iterator[DB]:
         actives = self.app.assignor.assigned_actives()
-        topic = self.table._changelog_topic_name()
+        topic = self.table.changelog_topic_name
```

Comment:
isn't this a function?

Reply:
the underscore version is. The non-underscore version is an `@property`